src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -739,7 +739,7 @@ public void TestStartWithMissingFile(bool fullPath)
             Assert.NotEqual(0, e.NativeErrorCode);
         }
 
-        [PlatformSpecific(TestPlatforms.Windows)]
+        [PlatformSpecific(TestPlatforms.Windows)]  // Needs permissions on Unix
```

Comment:
Checked again and the failure on non-root says needs permissions. Passes when run as root.